scripts/copy_demos.py

```diff
@@ -0,0 +1,59 @@
+import shutil
+import os
+import pathlib
+import textwrap
+import argparse
+
+
+def copy_all_demos(source_dir: str, dest_dir: str):
+    all_demos = []
+    for p in os.listdir(source_dir):
+        full_dir = os.path.join(source_dir, p)
+        is_dir = os.path.isdir(full_dir)
+        no_requirements = not os.path.exists(os.path.join(full_dir, "requirements.txt"))
+        has_app = os.path.exists(os.path.join(full_dir, "run.py"))
+        if is_dir and no_requirements and has_app:
+            all_demos.append(p)
+
+    to_exclude = [
```

Comment:
Why are we excluding these ones?

Reply:
If there's some sort of underlying issue affecting all of these demos, it might also affect new demos that we create in the future and break this script. So perhaps better to select a few demos to create instead.

Reply:
There was a lot of iteration here so I forgot exactly why each was excluded 😅 Will select a hard-coded list to avoid flakiness.